### PR TITLE
Windows flutter metrics

### PR DESCRIPTION
**Pull request by blagoev**
metrics on flutter windows

### CODE REVIEW

File: lib/src/cli/common/utils.dart [hunk 1]
```diff
@@ -54,10 +50,102 @@ FutureOr<T?> safe<T>(
...
-    }
-  });
+final log = createLogger();
+
+Logger createLogger() {
+  if (!Platform.isWindows) {
+    return Logger('metrics')
+      ..onRecord.listen((record) {
+        try {
+          stdout.writeln('[${record.level.name}] ${record.message}');
```
[the remark this answers is not in the document]
Comment (blagoev) [reply lib/src/cli/common/utils.dart:60]: I think  it fail above on the  `..onRecord.listen` call

[the remark this answers is not in the document]
Comment (nielsenko) [reply lib/src/cli/common/utils.dart:60]: hmm .. okay

**Review by nielsenko — APPROVED** [review]

File: lib/src/cli/common/utils.dart (hunk 1 above)
[the remark this answers is not in the document]
Comment (blagoev) [reply lib/src/cli/common/utils.dart:60]: hold on. it seems to work. will fix.

File: lib/src/cli/common/utils.dart [hunk 2]
```diff
@@ -53,99 +53,18 @@ FutureOr<T?> safe<T>(FutureOr<T> Function() f, {String message = 'Ignoring error
...
-          }
-        } catch (_) {}
-      });
-  }
-
-  return WindowsLogger('metrics');
+  return Logger('metrics')
+    ..onRecord.listen((record) {
+      try {
+        if (!Platform.isWindows) {
```
Comment (nielsenko) [inline lib/src/cli/common/utils.dart:59]:
```suggestion
        if (Platform.isWindows) {
```

File: lib/src/cli/common/utils.dart [hunk 3]
```diff
@@ -53,99 +53,18 @@ FutureOr<T?> safe<T>(FutureOr<T> Function() f, {String message = 'Ignoring error
...
+          print('[${record.level.name}] ${record.message}');
+          return;
```
Comment (nielsenko) [inline lib/src/cli/common/utils.dart:61]:
```suggestion
          if (record.error != null) {
            print(record.error);
          }
          return;
```

**Review by nielsenko — CHANGES_REQUESTED** [review]

File: lib/src/cli/common/utils.dart (hunk 1 above)
[the remark this answers is not in the document]
Comment (nielsenko) [reply lib/src/cli/common/utils.dart]: I have a few comments to the fix

File: lib/src/cli/common/utils.dart (hunk 2 above)
Comment (blagoev) [reply lib/src/cli/common/utils.dart]: good catch

File: lib/src/cli/common/utils.dart (hunk 3 above)
Comment (blagoev) [reply lib/src/cli/common/utils.dart:64]: fixed

**Review by nielsenko — APPROVED** [review]

**Pull request merged**